docs/projects/project-triggers/automatic-release-creation.md

```diff
@@ -4,27 +4,29 @@ description: Automatic release creation allows you to automatically create a new
 position: 15
 ---
 
-:::success
+:::hint
 **Consider using a build server extension**
-We have extensions/plugins available for the most popular build servers. These extensions will help you [create packages](/docs/packaging-applications/index.md), [push those packages to the built-in repository](/docs/packaging-applications/package-repositories/built-in-repository/index.md#pushing-packages-to-the-built-in-repository), create releases and deploy them to your environments:
+We have [extensions/plugins](/docs/packaging-applications/build-servers/index.md) available for the most popular build servers. These extensions will help you [create packages](/docs/packaging-applications/index.md), [push those packages to the built-in repository](/docs/packaging-applications/package-repositories/built-in-repository/index.md#pushing-packages-to-the-built-in-repository), create releases and deploy them to your environments:
 
-- [TeamCity](/docs/packaging-applications/build-servers/teamcity.md).
-- [Azure DevOps (formerly VSO) and TFS 2015](/docs/packaging-applications/build-servers/tfs-azure-devops/using-octopus-extension/index.md).
-  :::
+ - [AppVeyor](/docs/packaging-applications/build-servers/appveyor/index.md)
+ - [Azure DevOps & Team Foundation Server](/docs/packaging-applications/build-servers/tfs-azure-devops/index.md)
+ - [Bamboo](/docs/packaging-applications/build-servers/bamboo.md)
+ - [BitBucket Pipelines](/docs/packaging-applications/build-servers/bitbucket-pipelines/index.md)
+ - [Continua CI](/docs/packaging-applications/build-servers/continua-ci.md)
+ - [Jenkins](/docs/packaging-applications/build-servers/jenkins.md)
+ - [TeamCity](/docs/packaging-applications/build-servers/teamcity.md)
 
-:::success
-Take a look at the [end-to-end guide for Azure DevOps/TFS](/docs/packaging-applications/build-servers/tfs-azure-devops/index.md) which covers building and packaging your application, creating releases and deploying to your environments.
 :::
 
-:::hint
+## Getting started {#AutomaticReleaseCreation-Gettingstarted}
+
+If you use the [built-in Octopus package repository](/docs/packaging-applications/package-repositories/index.md), you can now select a package, that when uploaded it will automatically create a release.
```

Comment:
Slight tweak:
```suggestion
If you use the [built-in Octopus package repository](/docs/packaging-applications/package-repositories/index.md), you can now select a package, and when that package is uploaded it will automatically create a release.
```